releases/index.html

```diff
@@ -23,13 +24,59 @@
   | <a href="#14.2">14.2</a>
   | <a href="#14.1">14.1</a>
 </p>
-
 <div class="grid-10">
   <div class="version">
+    <a name="16.2"></a>
     <a name="latest"></a>
-
+    <h4>16.2 (latest)</h4>
+    <div class="section">
+      <h5>What's new in GoCD 16.2</h5>
+    <div class="section">
+      <h5>Features and bug fixes</h5>
+      <ul>
+        <li>
```

Comment:
Would it be interesting to mention the websocket work? Even if it is toggled off. A mention might be interesting. Or a "What's coming up?" section. Mentioning the elastic agent plugins too, maybe? Just thinking out loud.